### ISSUE TITLE
Deleting cluster states that kubeconfig isn't auto-generated

### ISSUE THREAD

**Issue by jpds** [weaveworks/eksctl#236]
**What happened?**

I ran the following with no kubeconfig at all present in my home directory:

```
$ mv .kube .kube-old
$ eksctl create cluster -r eu-west-1
$ eksctl delete cluster --region eu-west-1 --name exciting-rainbow-1538647578
2018-10-04T12:14:00+01:00 [ℹ]  deleting EKS cluster "exciting-rainbow-1538647578"
2018-10-04T12:14:00+01:00 [ℹ]  will delete stack "eksctl-exciting-rainbow-1538647578-nodegroup-0"
2018-10-04T12:14:00+01:00 [ℹ]  waiting for stack "eksctl-exciting-rainbow-1538647578-nodegroup-0" to get deleted
2018-10-04T12:15:47+01:00 [ℹ]  will delete stack "eksctl-exciting-rainbow-1538647578-cluster"
2018-10-04T12:15:48+01:00 [!]  as you are not using the auto-generated kubeconfig file you will need to remove the details of cluster exciting-rainbow-1538647578 manually
2018-10-04T12:15:48+01:00 [✔]  the following EKS cluster resource(s) for "exciting-rainbow-1538647578" will be deleted: node group, cluster. If in doubt, check CloudFormation console
```

**Versions**
Please paste in the output of these commands:
```
$ eksctl version
2018-10-04T14:01:18+01:00 [ℹ]  versionInfo = map[string]string{"builtAt":"2018-09-28T08:29:56Z", "gitCommit":"13286de1b06717200a5fbe692878be4937a1b05f", "gitTag":"0.1.4"}
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.2", GitCommit:"bb9ffb1654d4a729bb4cec18ff088eacc153c239", GitTreeState:"clean", BuildDate:"2018-08-07T23:17:28Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by richardcase** [comment]
@jpds - what would you expect to occur in this scenario? 

IIn my head,  if there is no default kubeconfig file and there isn't an autogenerated generated kubeconfig file then we should write a log file saying that we can't find a kubeconfig file and so no cleanup will be done.

**Comment by richardcase** [comment]
@jpds  - the autogenerated kube config file is stored in a subdirectory of the kube folder (normally ~/.kube). For example, `~/.kube/eksctl/clusters/test-cluster`.  In your scenario above you are deleting the `.kube` folder and so there is no kube config for it to delete. 

An autogenerated kubeconfig is deleted if it exists. Recently @kschumy added deletion of the cluster details from a shared kubeconfig file (see #226) as well.

**Comment by jpds** [comment]
@richardcase With `versionInfo = map[string]string{"builtAt":"2018-09-28T08:29:56Z", "gitCommit":"13286de1b06717200a5fbe692878be4937a1b05f", "gitTag":"0.1.4"}` - I do not have a subdirectory in `~/.kube/` - I can see my EKS cluster is directly in `config`.

Sorry if it wasn't clear, but I moved my working kubeconfig out of the way, *before* creating my EKS cluster, I then *removed* that EKS cluster without changing the config and eksctl told me that it couldn't remove the config as it wasn't autogenerated which it was.

**Comment by jpds** [comment]
Ah, the linked pull request appears to have been merged just after `0.1.4` was released. :)

**Comment by richardcase** [comment]
@jpds - ah yes, i misread the order, sorry. When 0.1.5 is out then this should handle this scenario.

**Comment by errordeveloper** [comment]
This was indeed fixed in 0.1.5.
